e2e/atlas/dbusers_test.go

```diff
@@ -159,22 +180,52 @@ func TestDBUsers(t *testing.T) {
 	})
 
 	t.Run("Delete", func(t *testing.T) {
-		cmd := exec.Command(cliPath,
-			atlasEntity,
-			dbusersEntity,
-			"delete",
-			username,
-			"--force",
-			"--authDB",
-			"admin")
-		cmd.Env = os.Environ()
-		resp, err := cmd.CombinedOutput()
+		testDeleteUser(t, cliPath, atlasEntity, dbusersEntity, username)
+		testDeleteUser(t, cliPath, atlasEntity, dbusersEntity, stdinUsername)
+	})
+}
 
-		if err != nil {
-			t.Fatalf("unexpected error: %v, resp: %v", err, string(resp))
-		}
+func testCreatePasswordCmd(t *testing.T, cmd *exec.Cmd, username string) {
+	t.Helper()
 
-		expected := fmt.Sprintf("DB user '%s' deleted\n", username)
-		assert.Equal(t, expected, string(resp))
-	})
+	cmd.Env = os.Environ()
+
+	resp, err := cmd.CombinedOutput()
+	a := assert.New(t)
+	a.NoError(err)
```

Comment:
[nit] you want `require` here so it doesn't run any other test
```suggestion
	require.NoError(t, err)
	a := assert.New(t)
```